src/errors/bad-request.ts

```diff
@@ -0,0 +1,8 @@
+import CustomAPIError from './custom-api-error'
+
+export default class BadRequestError extends CustomAPIError {
+  constructor (message: string) {
+    super(message)
+    this.statusCode = 404
```

Comment:
Are 404 is not a BadRequest error code?

Reply:
Yes, you're right, I didn't notice the typo😅